### ISSUE TITLE
Show list of user files

### ISSUE THREAD

**Issue by damascene** [nosdav/pastebin#16]
Currently there is no way to see the files the user own. User access to file is based on remembering the file name and pressing the load button. 

I suggest adding list of user files to be viewable and editable by them.

**Comment by melvincarvalho** [comment]
Good idea, although in a decentarlized world no one can ever know ALL the files you have, but that's good for privacy

I think I need to do directory listings, which was always something needed.  But I think we can do a bit more here, too.

**Comment by damascene** [comment]
My point is about the user ability to see the files he created. Not about allowing others to see what files a user have created. Maybe this could be an option if someone want to have their files public/unlisted/private
